tests/conftest.py

```diff
@@ -10,6 +10,19 @@
 # Ensure progress output even when not outputting to raw sys.stderr console
 os.environ["DVC_IGNORE_ISATTY"] = "true"
 
+REMOTES = {
+    # remote: enabled_by_default?
+    "azure": False,
```

Comment:
The `False`/`disabled` remotes does not run on normal `pytest .` cases. You have to either enable all of the remotes, with `--all` or enable some of them by using `--enable-azure`, for example.